Source/p_spec.c

```diff
@@ -163,6 +165,12 @@ void P_InitPicAnims (void)
       lastanim->istexture = animdefs[i].istexture;
       lastanim->numpics = lastanim->picnum - lastanim->basepic + 1;
 
+      // [crispy] add support for SMMU swirling flats
+      if (lastanim->speed > 65535 || lastanim->numpics == 1)
```

Comment:
Maybe we can add an option to enable swirling everywhere, I don't know of any PWAD that uses this. I thought it would work like in Doom Retro.

Reply:
Eternity has "r_swirl" console command wich enables it for all animated flats. Of course it may break some things, but currently no one will see this feature.

Reply:
Okay, we could add a feature switch to the General menu, next to the sky rendering options.

Reply:
Unfortunately there is no space left in General menu. I can make another rearrangement myself (add mouselook mode, bring back default complevel, restore two deleted items, maybe even add net_player_name). Or we should delete something.

Reply:
Yes, it's always awful to add that one feature that requires to open up a new menu page without having enough other menu items to at least half fill that new page. 😁

Reply:
We can port over the a11y options from Crispy and fill the new page with them.

Reply:
I had to add a third page to the General menu. Doesn't look too bad, but if you have any further ideas for another rearrangement, feel free to do so. :wink: